### PR TITLE
Postfix queue

### PR DESCRIPTION
**Pull request by stephdl**
Implement Postfix queue management UI

image
image
image
image
image
image
image

### CODE REVIEW

File: ui/public/i18n/en/translation.json [hunk 1]
```diff
@@ -331,5 +331,36 @@
...
+    "deferred_queue_details": "Details of the deferred email",
+    "queue_id": "Queue ID",
+    "arrival_time": "Arrival time",
+    "message_size": "Message size",
+    "sender": "Sender",
+    "recipient": "Recipient",
+    "delay_reason": "Delay reason",
+    "close": "Close",
+    "delete_queue": "Delete all emails",
+    "confirm_delete_all_queued_message": "All emails will be deleted, are you sure",
```
Comment (gsanchietti) [inline ui/public/i18n/en/translation.json:359]:
```suggestion
    "confirm_delete_all_queued_message": "All emails will be deleted, are you sure?",
```

File: ui/public/i18n/en/translation.json [hunk 2]
```diff
@@ -331,5 +331,36 @@
...
+    "no_queue": "No emails found",
```
Comment (gsanchietti) [inline ui/public/i18n/en/translation.json:360]:
```suggestion
    "no_queue": "No email found",
```

File: ui/public/i18n/en/translation.json [hunk 3]
```diff
@@ -331,5 +331,36 @@
...
+    "no_queue_description": "The email queue is empty.",
+    "delete_email": "Delete the email",
+    "confirm_delete_queued_message": "The email will be deleted, are you sure",
```
Comment (gsanchietti) [inline ui/public/i18n/en/translation.json:363]:
```suggestion
    "confirm_delete_queued_message": "This email will be deleted, are you sure?",
```

**Review by gsanchietti — APPROVED** [review]
It seems good to me, let's wait the UI part from Andrea

**Review by gsanchietti — COMMENTED** [review]
I know that the UI requires a little rework

File: ui/public/i18n/en/translation.json [hunk 4]
```diff
@@ -331,5 +333,38 @@
...
+    "delay_reason": "Delay reason",
+    "close": "Close",
+    "delete_queue": "Delete all emails",
+    "confirm_delete_all_queued_message": "All emails will be deleted, are you sure ?",
+    "no_queue": "No email found",
+    "no_queue_description": "The email queue is empty.",
+    "delete_email": "Delete the email",
+    "confirm_delete_queued_message": "The email will be deleted, are you sure ?",
+    "cancel": "Cancel",
+    "queue_tooltip": "Only failed attempts to be delivered are listed in the queue page.",
```
Comment (andre8244) [inline ui/public/i18n/en/translation.json:367]:
```suggestion
    "queue_tooltip": "These are the messages that are waiting to be relayed in the SMTP mail queue. In normal conditions, this queue should be empty or contain just a few messages.",
```
Taken from NS7 manual :wink:

File: ui/public/i18n/en/translation.json [hunk 5]
```diff
@@ -331,5 +333,38 @@
...
+    "sender": "Sender",
+    "recipient": "Recipient",
+    "delay_reason": "Delay reason",
+    "close": "Close",
+    "delete_queue": "Delete all emails",
+    "confirm_delete_all_queued_message": "All emails will be deleted, are you sure ?",
+    "no_queue": "No email found",
+    "no_queue_description": "The email queue is empty.",
+    "delete_email": "Delete the email",
+    "confirm_delete_queued_message": "The email will be deleted, are you sure ?",
```
Comment (andre8244) [inline ui/public/i18n/en/translation.json:365]:
```suggestion
    "confirm_delete_queued_message": "The email message will be deleted, are you sure?",
```

File: ui/public/i18n/en/translation.json [hunk 6]
```diff
@@ -331,5 +333,38 @@
...
+    "queue_id": "Message ID",
+    "arrival_time": "Arrival time",
+    "message_size": "Message size",
+    "sender": "Sender",
+    "recipient": "Recipient",
+    "delay_reason": "Delay reason",
+    "close": "Close",
+    "delete_queue": "Delete all emails",
+    "confirm_delete_all_queued_message": "All emails will be deleted, are you sure ?",
+    "no_queue": "No email found",
```
Comment (andre8244) [inline ui/public/i18n/en/translation.json:362]:
```suggestion
    "no_queue": "Mail queue is empty",
```

File: ui/src/views/DeferredQueue.vue [hunk 7]
```diff
@@ -0,0 +1,629 @@
...
+              @click="toggleDeleteQueueAll"
+              :disabled="loading.listDeferredQueue || loading.setDeleteQueue"
+              >{{ $t("queue.delete_all") }}
+            </NsButton>
+          </template>
+        </cv-column>
+      </cv-row>
+      <cv-row>
+        <cv-column>
+          <NsDataTable
```
Comment (andre8244) [inline ui/src/views/DeferredQueue.vue:68]: Let's add a `<cv-tile light>` between `<cv-column>` and `<NsDataTable>` to have a little more contrast on the page elements

**Pull request merged**